lib/token_store/sql_store.rb

```diff
@@ -28,12 +33,29 @@ def delete(token)
       record.destroy!
     end
 
+    def delete_all_for_user(userid)
+      user = find_user_by_userid(userid)
+      user.sessions.where(Session.arel_table[:session_id].matches("#{@namespace}%")).destroy_all
+    end
+
     private
 
     attr_reader :namespace
 
     def session_key(token)
       "#{namespace}:#{token}"
     end
+
+    def user_id(userid)
+      find_user_by_userid(userid).id
+    end
+
+    def find_user_by_userid(userid)
+      User.in_my_region.where('lower(userid) = ?', userid.downcase).first
+    end
+
+    def find_user_by_id(user_id)
```

Comment:
`by_id` implies the id column which could be confusing.

Additionally, creating find_by methods make rubocop think the callers are using dynamic find_by Rails methods, and it gets mad...sucks, but we've been using lookup_by instead to avoid that.

Reply:
Oh I'm sorry...I think the `user_id` was confusing because I conflated it with `userid`...naming is hard.

```suggestion
    def find_user_by_id(id)
```

Though I'm not sure what to do about the `user_id` method